src/main/java/com/google/sps/servlets/NewsServlet.java

```diff
@@ -30,11 +30,11 @@
 public class NewsServlet extends HttpServlet {
 
   private final String API_KEY = "test"; // Insert actual API key when testing.
-  private int numArticlesPerKeyword = 3;
+  private int NUM_ARTICLES_PER_KEYWORD = 3;
```

Comment:
Since it's a constant, I recommend putting the _final_ keyword here. So:
`private final int NUM_ARTICLES_PER_KEYWORD = 3;`